.ci/run-clang-tidy.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
```

Comment:
Am I correct that this file was taken directly from LLVM? Does it really only work with python2 or it is possible to change it to pyhton3?

Reply:
Sorry, I didn't notice
```
python3 ../.ci/run-clang-tidy.py -quiet -header-filter=../src/DynamicRupture/*
```